flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/DescribeOperator.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.operations;
+
+import org.apache.flink.annotation.Internal;
+
+/**
+ * An {@link Operation} that describes the catalog, database and table, e.g. DESCRIBE CATALOG
+ * catalogName, DESCRIBE DATABASE [catalogName.]dataBaseName and DESCRIBE TABLE [[catalogName.]
+ * dataBaseName.]tableName.
+ */
+@Internal
+public interface DescribeOperator extends Operation, ExecutableOperation {}
```

Comment:
Nit: I think the naming `DescribeOperator` needs to be reconsidered. All the rest Operations are named with `XXOperation`.